### PR TITLE
CHAOS-5499 Linux JVM CPU stress

### PR DESCRIPTION
**Pull request by SmritiSatya**
Linux JVM CPU stress

### CODE REVIEW

File: docs/chaos-engineering/chaos-faults/linux/linux-jvm-cpu-stress.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+JVM chaos faults use the [Byteman utility](https://byteman.jboss.org/) to inject chaos faults into the JVM.
+:::
+
+![Linux JVM CPU stress](./static/images/linux-jvm-cpu-stress.png)
+
+## Use cases
+Linux JVM CPU stress:
+- Tests the system's ability to handle high payloads.
+- Evaluates the application's behavior in high-stress cases.
+- Induces CPU consumption and exhaustion on the target Linux machines.
```
Comment (neelanjan00) [inline docs/chaos-engineering/chaos-faults/linux/linux-jvm-cpu-stress.md:21]:
```suggestion
- Induces CPU consumption and exhaustion on the target Java process JVM.
```

File: docs/chaos-engineering/chaos-faults/linux/linux-jvm-cpu-stress.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+kind: LinuxFault
+metadata:
+  name: linux-jvm-cpu-stress
+  labels:
+    name: jvm-cpu-stress
+spec:
+  jvmChaos/inputs:
+    duration: 30s
+    port: 9091
+    pid: 0
```
Comment (neelanjan00) [inline docs/chaos-engineering/chaos-faults/linux/linux-jvm-cpu-stress.md:95]: We aren't using PID here, please check

Comment (neelanjan00) [reply docs/chaos-engineering/chaos-faults/linux/linux-jvm-cpu-stress.md:95]: Please check in other faults too.

Comment (SmritiSatya) [reply docs/chaos-engineering/chaos-faults/linux/linux-jvm-cpu-stress.md:95]: https://github.com/wings-software/enterprise-chaos-hub/blob/main/faults/linux/linux-jvm-cpu-stress/fault.yaml I referred this YAML. Please confirm if that is inconsistent.

**Review by neelanjan00 — APPROVED** [review]

**Pull request merged**